File: solutions/cpp/2215-find-the-difference-of-two-arrays.cpp
class Solution {
public:
    vector<vector<int>> findDifference(vector<int>& nums1, vector<int>& nums2) {
        set<int> set1(nums1.begin(), nums1.end());
        set<int> set2(nums2.begin(), nums2.end());
        
        vector<int> diff1, diff2;
        for(int num : set1)
            if(set2.find(num) == set2.end())
                diff1.push_back(num);
                
        for(int num : set2)
            if(set1.find(num) == set1.end())
                diff2.push_back(num);
                
        return {diff1, diff2};
    }
};

File: solutions/python/2215-find-the-difference-of-two-arrays.py
class Solution:
    def findDifference(self, nums1: List[int], nums2: List[int]) -> List[List[int]]:
        set1 = set(nums1) 
        set2 = set(nums2) 
        diff1 = set1 - set2 
        diff2 = set2 - set1 
        return [list(diff1), list(diff2)] 
                                           
